themes/helm/layouts/partials/banner.html

```diff
@@ -0,0 +1,16 @@
+<div id="banner">
+  <div class="cc-container">
+
+    <!-- viewing helm 2 version of site (desktop) -->
+    <p class="center text-center show-for-medium-up">You are viewing Helm 2 (latest stable). &nbsp; Helm 3 is coming soon (alpha-1 is <a href="https://v3.helm.sh"><strong>here</strong>). &nbsp; Visit the <a href="https://v3.helm.sh"><strong>v3 docs</strong></a> or read the <a href="https://helm.sh/blog"><strong>blog</strong></a> for details.</p>
```

Comment:
What do you think about changing the verbiage to "3.0.0-alpha.1 is here"? Is that too long for a banner?

Reply:
Yeah that makes sense, and is fine in length. Would you rather it as: 

```
You are viewing Helm 2 (latest stable).   Helm 3 is coming soon (3.0.0-alpha.1 is here) [...]
```

or 

```
You are viewing Helm 2 (latest stable).   Helm 3.0.0-alpha.1 is here [...]
```

Reply:
I like the latter